src/layers/normalise.jl

```diff
@@ -55,6 +55,9 @@ function _dropout_mask(rng, x, p; dims=:)
   return y
 end
 
+# TODO move this to NNlib
+Zygote.ChainRulesCore.@non_differentiable dropout_mask(rng, x, p)
```

Comment:
```suggestion
Zygote.ChainRulesCore.@non_differentiable dropout_mask(::Any, ::Any, ::Any)
```
I noticed #1863 uses this signature. Is there a functional difference between the two?